packages/shared/src/components/MainLayout.tsx

```diff
@@ -52,11 +55,18 @@ export default function MainLayout({
   const { windowLoaded } = useContext(ProgressiveEnhancementContext);
   const { user, showLogin, loadingUser } = useContext(AuthContext);
   const [showGreeting, setShowGreeting] = useState(false);
+  const showSidebar = useMedia(
+    [footerNavBarBreakpoint.replace('@media ', '')],
+    [true],
+    false,
+  );
```

Comment:
This seems to be repeated.
Please consider creating a hook for this.

Another suggestion would be to use some sort of context to reduce the number of event listeners. But not sure if we should really care about it.